lib/engine/game/base.rb

```diff
@@ -1436,6 +1436,14 @@ def round_description(name, round_number = nil)
         description.strip
       end
 
+      def can_select?(_entity)
```

Comment:
why do you need can_select?

Reply:
I needed a way to prevent parring a corporation that is in an unavailable layer (1860 specific thing). If can_select? returns false, then the corp isn't selectable in the list. I thought this was a cool way of doing it.  The alternative was to add a can_ipo? or similar qualifier to this line:
`children << render_input if @selected_corporation == corporation`

Since I didn't want to modify all the game-specific steps that use the UI:Round, I had to made it part of game/base.

I can switch to do the latter, or give the method a different name.

Reply:
Ok, switching to the approach where the corp is selectable, but it won't have any par buttons rendered unless @game.corporation_available? returns true (which it will in the game/base definition).